Applications/Games/Hearthstone/Online/script.js

```diff
@@ -1,22 +1,24 @@
 include(["engines", "wine", "quick_script", "online_installer_script"]);
-include(["engines", "wine", "plugins", "windows_version"]);
 include(["engines", "wine", "verbs", "vcrun2015"]);
 include(["engines", "wine", "verbs", "corefonts"]);
 
+// BUG: https://github.com/PhoenicisOrg/scripts/issues/792
+
+// TODO: OSX support, needs verification
```

Comment:
Please do not add these kind of comments on the code

Reply:
I've found similar kind of comment in the `Europa Universalis 2` script.

Reply:
Yep, I think I will just add a link to the website where the patches can be downloaded ^^.

Reply:
No there is a website with static links to those patches but I do not own that game. Anyway the reason I've pointed that out was because I was wondering if the comments in that script should also be removed.

Reply:
Using comments in installers seems sane assuming they are in testing. -> Helps with script development. 

I would agree with not using them in non-testing scripts.